src/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -822,7 +823,11 @@ def custom_forward(*inputs):
             all_hidden_states = all_hidden_states + (hidden_states,)
 
         if not return_dict:
-            return tuple(v for v in [hidden_states, presents, all_hidden_states, all_self_attentions] if v is not None)
+            return tuple(
+                v
+                for v in [hidden_states, presents, all_hidden_states, all_self_attentions, all_cross_attentions]
+                if v is not None
+            )
```

Comment:
Oh that's a good catch

Reply:
Regarding this, feel free to open a PR with the fix; that's definitely a bug that you've found!